Insterm/SSHTerminalView.swift

```diff
@@ -38,6 +39,28 @@ class SSHTerminalView: TerminalView, TerminalViewDelegate, NMSSHChannelDelegate
         print("didReadError: \(error)")
     }
     
+    func isConnected() -> Bool {
+        if (connection != nil) {
+            return connection!.isConnected()
+        } else {
+            return false
+        }
+    }
+    
+    // ピンチイン・アウトでフォントサイズを変更
+    @objc func pinchInOut(_ gestureRecognizer: UIPinchGestureRecognizer) {
+        if gestureRecognizer.state == .began || gestureRecognizer.state == .changed {
```

Comment:
guardを使うとブロックのネストを減らすことができるので、保守性が上がります。

```suggestion
        guard if gestureRecognizer.state == .began || gestureRecognizer.state == .changed else { return }
```